core/tests/test_common.py

```diff
@@ -0,0 +1,16 @@
+import geopandas
+import momepy
+
+import core
+
+
+class TestContinuity:
+    def setup_method(self):
+        self.city = "Liège"
+        self.roads = momepy.remove_false_nodes(core.utils.read_parquet_roads(self.city))
```

Comment:
Can we just read the cleaned one? This may take a bit.

Reply:
I'll see about taking care of these when back from vacation.

Reply:
done in [a9e8f83](https://github.com/uscuni/simplification/pull/80/commits/a9e8f83c5289b78b8bc78df1729c8e846acdca44)